app/controllers/users_controller.rb

```diff
@@ -1,3 +1,5 @@
+include AccessTokenHelper
+
```

Comment:
Any reason why this isn't mixed into the class? Not even sure what's the right approach or if it even matters.